components/feature/accounts-push/src/main/java/mozilla/components/feature/accounts/push/FxaPushSupportFeature.kt

```diff
@@ -157,7 +157,9 @@ internal class AccountObserver(
         // registration could happen after onDevicesUpdate has been called, without having to tie this
         // into the account "auth lifecycle".
         // See https://github.com/mozilla-mobile/android-components/issues/8766
-        account.deviceConstellation().registerDeviceObserver(constellationObserver, lifecycleOwner, autoPause)
+        CoroutineScope(Dispatchers.Main).launch {
```

Comment:
nit: If you do not keep a reference to the scope, to limit its lifetime, then you could use `GlobalScope`:

```GlobalScope.launch(Dispatchers.Main) {  }```

Reply:
What I do like about this is that all instances of this pattern are then clearly identified by usages of `GlobalScope`. Makes it easy to find and reassess if we need to manage these scopes better.